src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,80 @@
+package duke.command;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+import duke.List;
+import duke.Storage;
+import duke.Ui;
+import duke.task.Tasks;
+
+/**
+ * Represents a Command which adds a Task to the task List.
+ */
+public class AddCommand extends Command {
+
+    private Tasks taskType;
+    private String description;
+    private String time;
+    private LocalDateTime localDateTime;
+
+    /**
+     * Constructor for a AddCommand, specifically used for a Event Task.
+     *
+     * @param taskType TaskType of the Task to be added.
+     * @param description Description of the Task to be added.
+     * @param time Time the Task added occurs at.
+     */
+    public AddCommand(Tasks taskType, String description, String time) {
+        super(false);
+        this.taskType = taskType;

```

Comment:
May be simpler if you call 
```
this.AddCommand(taskType, description);
```
instead of typing out the whole of line 30 and 31?